### PR TITLE
Update Upgrade Documentation for all Installation Methods

### LINKED ISSUE #1730: Upgrading Operator?
** Which example are you working with? **
I installed the operator using the instructions here: https://access.crunchydata.com/documentation/postgres-operator/4.4.0/installation/postgres-operator/ (well version 4.3.2 last week) and figured I'd update to 4.4.0, however I cannot find any upgrade guidelines that match those installation instructions.

As I understand there are several installation options: bash, ansible, helm and the "quickstart". I can find upgrade instructions for the "quickstart" installation type. In general is there an installation type that is more commonly used/recommended?

### PR DESCRIPTION
**Pull request by tjmoore4**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change, as applicable?
 - [x] Have you tested your changes on all related environments with successful results, as applicable?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
Upgrade docs do not explicitly cover all current installation methods.

**What is the new behavior (if this is a feature change)?**
This update adds additional information for how to upgrade the
Crunchy PostgreSQL Operator using the PostgreSQL Operator Installer
and Helm Chart installation methods.

**Other information**:

closes #1730

### CODE REVIEW

File: docs/content/Upgrade/automatedupgrade.md [hunk 1]
```diff
@@ -35,15 +35,31 @@ The automated upgrade procedure is designed to facilate the quickest and most ef
 
 ##### NOTE: As with any upgrade procedure, it is strongly recommended that a full logical backup is taken before any upgrade procedure is started. Please see the [Logical Backups](/pgo-client/common-tasks#logical-backups-pg_dump--pg_dumpall) section of the Common Tasks page for more information.
 
-### Automated Upgrade when using an Ansible installation of the PostgreSQL Operator
+### Automated Upgrade when using an Ansible installation, the PostgreSQL Operator Installer or the Helm Chart Installation of the PostgreSQL Operator
```
Comment (jkatz) [inline docs/content/Upgrade/automatedupgrade.md:38]:
```suggestion
### Automated Upgrade when using the PostgreSQL Operator Installer (`pgo-deployer`), Helm, or Ansible Ansible
```

File: docs/content/Upgrade/automatedupgrade.md [hunk 2]
```diff
@@ -35,15 +35,31 @@ The automated upgrade procedure is designed to facilate the quickest and most ef
...
+
+For existing PostgreSQL Operator deployments that were installed using Ansible, you will first need to check out the appropriate release tag of the Operator. Then please follow the [Update Instructions]({{< relref "installation/other/ansible/updating-operator.md" >}}), being sure to update the new inventory file with your required settings.
+
+#### PostgreSQL Operator Installer
+
+For existing PostgreSQL Operator deployments that were installed using the PostgreSQL Operator Installer, you will check out the appropriate release tag and update your the new configuration files. After this, you will need to update your Operator installation using the `DEPLOY_ACTION` method described in the [Configuring to Update and Uninstall](/installation/postgres-operator#configuring-to-update-and-uninstall) section of the documentation.
+
+Please note, you will need to ensure that you have executed the [post-installation cleanup](/installation/postgres-operator#post-installation) between each `DEPLOY_ACTION` activity.
+
+#### Helm
```
Comment (jkatz) [inline docs/content/Upgrade/automatedupgrade.md:56]: the order of these sections should be 

- `#### PostgreSQL Operator Installer`
- `#### Helm`
- `#### Ansible`

File: docs/content/Upgrade/automatedupgrade.md (hunk 1 above)
Comment (tjmoore4) [reply docs/content/Upgrade/automatedupgrade.md:38]: updated, thanks

File: docs/content/Upgrade/automatedupgrade.md (hunk 2 above)
Comment (tjmoore4) [reply docs/content/Upgrade/automatedupgrade.md:56]: updated.

**Pull request merged**